core/config/uniStrategy.js

```diff
@@ -0,0 +1,75 @@
+import harden from '@agoric/harden';
+
+import { insist } from '../../util/insist';
+import {
+  sameStructure,
+  mustBeSameStructure,
+  mustBeComparable,
+} from '../../util/sameStructure';
+
+const makeUniStrategy = descriptionCoercer => {
+  const uniStrategy = harden({
+    insistKind: optDescription => {
+      if (optDescription === null) {
+        return null;
+      }
+      insist(
+        !!optDescription,
+      )`Uni optDescription must be either null or truthy ${optDescription}`;
```

Comment:
add markers to demarcate ${optDescription}. quotes, angle brackets, parens, a prefix colon, or whatever.

Reply:
good idea